### PR TITLE
[pt] Add CONCORDANCIA_DAR_HORAS rule

### PR DESCRIPTION
**Pull request by p-goulart**
For #8945, part of languagetooler-gmbh/languagetool-premium#6261.

### CODE REVIEW

File: languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/grammar.xml [hunk 1]
```diff
@@ -887,6 +887,108 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA.
             </rule>
         </rulegroup>
 
+        <rulegroup id="CONCORDANCIA_DAR_HORAS" name="Concordância de construções como 'deram dez horas' => 'deu dez horas'" default="temp_off">
+            <rule> <!-- deu 10 horas -->
+                <pattern>
+                    <token postag="SENT_START"/> <!-- keep it tightly scoped -->
```
Comment (susanaboatto) [inline languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/grammar.xml:893]: In the future, we could add commas and `e` here as well

File: languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/grammar.xml [hunk 2]
```diff
@@ -887,6 +887,108 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA.
...
+                    <token postag_regexp="yes" postag="R." min="0" max="2"/>
+                    <marker>
+                        <token postag_regexp="yes" postag="V...3S." inflected="yes" regexp="yes">dar|bater</token>
+                    </marker>
+                    <token postag_regexp="yes" postag="Z.+"/>
+                    <token regexp="yes">horas|h(rs?)?</token> <!-- account for incorrect abbreviations as well -->
```
Comment (susanaboatto) [inline languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/grammar.xml:899]: It is technically incorrect, but perhaps XX:XX should be accounted for as well?

**Review by susanaboatto — APPROVED** [review]

File: languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/grammar.xml (hunk 2 above)
Comment (p-goulart) [reply languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/grammar.xml:899]: Did it in the other sub-rule – the tokeniser keeps `hh:mm` as a single token.

File: languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/grammar.xml (hunk 1 above)
Comment (p-goulart) [reply languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/grammar.xml:893]: I suppose, but for now I want to keep it extra safe, see how it goes.

**Pull request merged**